Mathlib/Init/Data/List/Instances.lean

```diff
@@ -76,14 +76,17 @@ instance decidableBex : ∀ (l : List α), Decidable (∃ x ∈ l, p x)
         cases' mem_cons.1 hm with h h
         · rw [h] at hp; contradiction
         · exact absurd ⟨y, h, hp⟩ h₂
-#align list.decidable_bex List.decidableBex
+#align list.decidable_bex List.decidableExists
 
-instance decidableBall (l : List α) : Decidable (∀ x ∈ l, p x) :=
+instance decidableForall (l : List α) : Decidable (∀ x ∈ l, p x) :=
   match (inferInstance : Decidable <| ∃ x ∈ l, ¬ p x) with
   | isFalse h => isTrue fun x hx => match ‹DecidablePred p› x with
     | isTrue h' => h'
     | isFalse h' => False.elim <| h ⟨x, hx, h'⟩
   | isTrue h => isFalse <| let ⟨x, h, np⟩ := h; fun al => np (al x h)
-#align list.decidable_ball List.decidableBall
+#align list.decidable_ball List.decidableForall
+
+@[deprecated] alias List.decidableBex := List.decidableExists -- 2024-03-23
+@[deprecated] alias List.decidableBall := List.decidableForall -- 2024-03-23
```

Comment:
I don't think we should be deprecating instance names.

Reply:
Makes sense! Dropped.